<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneratePortfolioForm</class>
 <widget class="QWidget" name="GeneratePortfolioForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1174</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="chooseAccount_Label">
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>10000</height>
          </size>
         </property>
         <property name="text">
          <string>选择账户：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="chooseAccount_ComboBox">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="strategy_Label">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>策略信息</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="geneBuySalePortfolio">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>生成组合</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="buySalePortfolio_Label">
       <property name="text">
        <string>对应账户买组合：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QTableView" name="strategy_table"/>
       </item>
       <item row="0" column="0">
        <widget class="QTableView" name="account_table"/>
       </item>
       <item row="0" column="2">
        <widget class="QTableView" name="buySalePortfolio_table"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="programInfo_Label">
       <property name="text">
        <string>------ 程序运行信息 -----</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="programInfo_Table"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
